router/image/templates/nginx.conf

```diff
@@ -292,22 +292,15 @@ http {
 
 ## start builder
 {{ if .deis_builder_host }}
-tcp {
-    access_log /opt/nginx/logs/git.log;
-    tcp_nodelay on;
-    timeout {{ or (.deis_router_builder_timeout_tcp) "1200000" }};
-
-    # same directive names, but these are in miliseconds...
-    proxy_connect_timeout       {{ or (.deis_router_builder_timeout_connect) "10000" }};
-    proxy_send_timeout          {{ or (.deis_router_builder_timeout_send) "1200000" }};
-    proxy_read_timeout          {{ or (.deis_router_builder_timeout_read) "1200000" }};
-
+stream {
     upstream builder {
         server {{ .deis_builder_host }}:{{ .deis_builder_port }};
     }
 
     server {
         listen 2222;
+        proxy_connect_timeout  {{ or (.deis_router_builder_timeout_connect) "10000" }};
+        proxy_timeout          {{ or (.deis_router_builder_timeout_tcp) "1200000" }};
```

Comment:
I think @bacongobbler was referring to `/deis/router/builder/timeout/tcp`, but it already exists: http://docs.deis.io/en/latest/customizing_deis/router_settings/